services/user.py

```diff
@@ -0,0 +1,31 @@
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    user = User.objects.create_user(username=username, password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
```

Comment:
same

Reply:
Btw, maybe better idea is to push declaration of this method on the start of the list and use implemented method on the remainings, since we already invented custom get_user with get_user_model() for every custom user?